lib/common.ts

```diff
@@ -8,7 +8,6 @@
 
 import * as rdf from 'rdf-js';
```

Comment:
```suggestion
import * as rdf from '@rdfjs/types';
```
This change will need to be made anywhere you use rdf-js.